scripts/dockerfiles/Dockerfile.metrics

```diff
@@ -6,17 +6,15 @@ WORKDIR /go/src/github.com/aws/amazon-vpc-cni-k8s
 ENV GO111MODULE=on
 ENV GOPROXY=direct
 
-# Copy modules in before the rest of the source to only expire cache on module
-# changes:
+# Copy modules in before the rest of the source to only expire cache on module changes:
 COPY go.mod go.sum ./
 RUN go mod download
 
 COPY . ./
 RUN make build-metrics
 
-FROM public.ecr.aws/amazonlinux/amazonlinux:2
-RUN yum update -y && \
-	yum clean all
+# Build from EKS minimal base + glibc
+FROM public.ecr.aws/eks-distro-build-tooling/eks-distro-minimal-base-glibc:latest.2
```

Comment:
Let's make sure that we can rely on `latest` tag.

Reply:
Spoke offline and the `latest` and `latest.2` tags are currently equivalent. In the future, `latest.2` may stay slightly back-level of `latest`, as `latest.2` is supposed to mean "latest stable AL2 build". Therefore, keeping `latest.2` tag.